README.md

```diff
@@ -177,12 +175,6 @@ Dashboard >> Settings >> Account
 | Name | Description |
 | ------------- | ------------- |
 | GOOGLE_ANALYTICS_ID | The ID of your Google Analytics account. |
-| GOOGLE_OAUTH_APP_NAME | The name of the Google application in the Google API console. |
```

Comment:
Restore this as these are relevant.  (The client has already provided them and I'd like to keep them in the ENV store so that when we do get this feature working we don't need to go back and ask them for those values)

Reply:
do you mean to restore just this line that you are pointing to or the whole thing